mod/site_notifications/start.php

```diff
@@ -114,6 +118,35 @@ function site_notifications_send($hook, $type, $result, $params) {
 	}
 }
 
+/**
+ * Fixes unwanted menu items on the entity menu
+ *
+ * @param \Elgg\Hook $hook Hook
+ *
+ * @return void|\ElggMenuItem[]
+ */
+function site_notifications_register_entity_menu(\Elgg\Hook $hook) {
```

Comment:
make this `Hook` or remove the `use` at the top of the file ;)

Reply:
removed the `use`